developer-docs-site/docs/nodes/full-node/aptos-db-restore.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Bootstrap Fullnode from Backup"
+slug: "aptos-db-restore"
```

Comment:
let's make a principle to not use slugs moving forward -- let the file names be the path, otherwise it gets a bit confusing imo.